src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/DateTimeToArrayTransformerTest.php

```diff
@@ -116,6 +116,30 @@ public function testTransformDifferentTimezones()
         $this->assertSame($output, $transformer->transform($input));
     }
 
+    public function testTransformDateTimeImmutable()
+    {
+        if (PHP_VERSION_ID < 50500) {
+            $this->markTestSkipped('DateTimeImmutable was introduced in PHP 5.5.0');
+        }
+
+        $transformer = new DateTimeToArrayTransformer('America/New_York', 'Asia/Hong_Kong');
+
+        $input = new \DateTimeImmutable('2010-02-03 04:05:06 America/New_York');
+
+        $dateTime = new \DateTimeImmutable('2010-02-03 04:05:06 America/New_York');
```

Comment:
@dosten I agree. I just copied the test function right above. Changing it would bring the two test functions out of sync. Do you think that is better?